### PR TITLE
Product Page

### PR DESCRIPTION
**Pull request by sahiba0915**
hello team, please review my code
-added page structure
-added styling to the page
-linked all the pages

Live preview: https://stylevilla-product.netlify.app/product.html

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+}
+
+.price #strike {
+  text-decoration: line-through;
+  font-size: 1.6rem;
+  color: #9a9a9a;
+}
+
+.discount {
+  color: #9a9a9a;
```
Comment (akash-1318) [inline css/cart.css]: Use color variables here.

**Review by chanchal-panpaliya — APPROVED** [review]
Looks Good To Merge

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.quantity {
+  font-size: 2rem;
+}
+
+#quantityno {
+  border: 0.1rem black solid;
```
Comment (akash-1318) [inline css/cart.css]: it should be solid black

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  width: 20%;
+  padding-left: 1rem;
+}
+
+.btn .remove {
+  width: 8rem;
+  height: 2.5rem;
+}
+
+.btn .move {
```
Comment (akash-1318) [inline css/cart.css]: here you are using the same code for different classes.

File: css/product.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+@import url("/style.css");
+
+.main-content {
+  display: grid;
+  margin: 0;
+  color: var(--secondary-color);
+  grid-template-columns: 1fr 3fr;
+  grid-template-rows: 1fr;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
```
Comment (akash-1318) [inline css/product.css]: here you can use grid-gap : 0px

File: css/product.css [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+  background-color: #f7f7f7;
```
Comment (akash-1318) [inline css/product.css]: use color variables

File: css/product.css [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+  color: var(--default-color);
+}
+
+.info {
+  margin-bottom: 2rem;
+}
+
+.product-card {
+  margin-bottom: 2rem;
+  border: 0.1rem solid rgba(0, 0, 0, 0.3);
```
Comment (akash-1318) [inline css/product.css]: you are using all three types to provide colors - hex code , name and rgba . Use only one type.

File: product.html [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+    <div>
+     <label><input type="radio" />Price- Low to High</label>
+    </div>
+    <div>
+     <label><input type="radio" />Price- High to Low</label>
+    </div>
+   </div>
+ 
+  
+
```
Comment (akash-1318) [inline product.html]: use proper formatting here , remove additional lines.

**Review by akash-1318 — COMMENTED** [review]
Added some points just look into these points else everything looks good from my side really clean code.

File: css/cart.css (hunk 1 above)
Comment (sahiba0915) [reply css/cart.css]: done

File: css/cart.css (hunk 2 above)
Comment (sahiba0915) [reply css/cart.css]: done

File: css/cart.css (hunk 3 above)
Comment (sahiba0915) [reply css/cart.css]: done

File: css/product.css (hunk 4 above)
Comment (sahiba0915) [reply css/product.css]: done

File: css/product.css (hunk 5 above)
Comment (sahiba0915) [reply css/product.css]: done

File: css/product.css (hunk 6 above)
Comment (sahiba0915) [reply css/product.css]: done

File: product.html (hunk 7 above)
Comment (sahiba0915) [reply product.html]: done

**Pull request merged**